src/routes/networks.cpp

```diff
@@ -0,0 +1,29 @@
+#include "networks.h";
```

Comment:
here should be no semicolon
```suggestion
#include "networks.h"
```